recent 2.13.x change broke compiler bridge in scala-dist-smoketest

at e.g. https://travis-ci.org/scala/scala-dist-smoketest/builds/483980520 we see

```
/tmp/sbt_c8e64c05/xsbt/ExtractAPI.scala:334: error: value isLocal is not a member of ExtractAPI.this.global.Symbol
      !(s.isPrivate && (s.privateWithin == NoSymbol || s.isLocal))
                                                         ^
one error found
```

looks like the relevant commit is scala/scala@5b8f9280de23d29ae960329cffc3a2d9a03f3eae, /cc @joroKr21

--- a/src/reflect/scala/reflect/internal/Symbols.scala
+++ b/src/reflect/scala/reflect/internal/Symbols.scala
@@ -1036,6 +1036,11 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
     /** Is this symbol owned by a package? */
     final def isTopLevel = owner.isPackageClass
 
+    // shim for sbt's compiler interface
+    /** Is this symbol defined in a block? */
+    @deprecated("use isLocalToBlock instead", "2.11.0")
+    final def isLocal: Boolean = owner.isTerm
+
     /** Is this symbol defined in a block? */
     final def isLocalToBlock: Boolean = owner.isTerm
 

